File: PASCAL/Cap03/c03ex05.pas
program c03ex05;

var
  NOME: String;

begin
  Write('Entre seu nome: ');
  ReadLn(NOME);

  WriteLn('Ola, ', NOME, '! Bem-vindo ao estudo de Pascal.');
  
  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: GO/Cap03/c03ex05.go
// c03ex05.go

package main
import (
  "fmt"
)

func main() {
  var NOME string

  fmt.Print("Entre seu nome: ")
  fmt.Scan(&NOME)

  fmt.Printf("Ola, %s! Bem-vindo ao estudo de Go.\n", NOME)

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
